bot/exts/easter/save_the_planet.py

```diff
@@ -0,0 +1,28 @@
+import json
+import random
+from pathlib import Path
+
+from discord import Embed
+from discord.ext import commands
+
+
+class SaveThePlanet(commands.Cog):
+    """A cog that teaches users how they can help our planet."""
+
+    json_embeds = []
+
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot = bot
+        with open(Path("bot/resources/save_the_planet.json"), 'r', encoding='utf8') as f:
+            for embed in json.load(f):
+                self.json_embeds.append(embed)
+
+    @commands.command(aliases=('savetheearth', 'saveplanet', 'saveearth'))
+    async def savetheplanet(self, ctx: commands.Context) -> None:
+        """Responds with a random tip on how to be ecofriendly and help our planet."""
+        await ctx.send(embed=Embed.from_dict(random.choice(self.json_embeds)))
```

Comment:
```suggestion
        return_embed = Embed.from_dict(random.choice(self.json_embeds))
        await ctx.send(embed=return_embed)
```
I think splitting this up into two lines makes it a bit more readable and easy to understand. If you don't like the variable name `return_embed`, feel free to choose your own.

Reply:
Ok, I wasn't sure about that either.